Tooling-Landscape/CapabilityMap/ocToolingAPI_work.yaml

```diff
@@ -180,6 +180,85 @@ paths:
         - casedatastore_auth:
             - write:scan
             - ''
+  /snippet-scan/:
+    get:
+      tags:
+        - snippetScanner
+      summary: Provide URL from which to pull sources for scanning.
+      operationId: uploadSources
+      parameters:
+        - name: purl
```

Comment:
Would it make sense to have a location instead of purl?
we might allow a file:// or s3:// or all sorts of locations